controllers/provider-openstack/charts/internal/cloud-provider-config/templates/cloud-provider-config.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gardener-extension-provider-openstack-configmap
+  namespace: {{ .Release.Namespace }}
+data:
+  cloudprovider.conf: |
+    [Global]
+    auth-url: {{ .Values.authUrl }}
+    domain-name: {{ .Values.domainName }}
+    tenant-name: {{ .Values.tenantName }}
+    username: {{ .Values.username }}
+    password: {{ .Values.password }}
+    [LoadBalancer]
+    lb-version=v2
+    lb-provider: {{ .Values.lbProvider }}
+    floating-network-id: {{ .Values.floatingNetworkID }}
+    subnet-id: {{ .Values.subnetID }}
+    create-monitor=true
+    monitor-delay=60s
+    monitor-timeout=30s
+    monitor-max-retries=5
+    {{- if and (semverCompare ">= 1.10.1" .Values.kubernetesVersion) (semverCompare "< 1.10.3" .Values.kubernetesVersion) }}
```

Comment:
Not sure if we should support Kubernetes 1.10.x in extension controllers, let's ask.

Reply:
Based on the discussion in Slack, version 1.10 will continue to be supported.